app/serializers/decidim/reporting_proposals/proposal_serializer_override.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ReportingProposals
+    module ProposalSerializerOverride
+      extend ActiveSupport::Concern
+
+      included do
+        include ActionView::Helpers::DateHelper
+
+        def serialize
+          {
+            id: proposal.id,
+            category: {
+              id: proposal.category.try(:id),
+              name: proposal.category.try(:name) || empty_translatable
+            },
+            scope: {
+              id: proposal.scope.try(:id),
+              name: proposal.scope.try(:name) || empty_translatable
+            },
+            participatory_space: {
+              id: proposal.participatory_space.id,
+              url: Decidim::ResourceLocatorPresenter.new(proposal.participatory_space).url
+            },
+            component: { id: component.id },
+            title: proposal.title,
+            body: proposal.body,
+            address: proposal.address,
+            latitude: proposal.latitude,
+            longitude: proposal.longitude,
+            state: proposal.state.to_s,
+            reference: proposal.reference,
+            answer: ensure_translatable(proposal.answer),
+            time_elapsed_to_answer: time_elapsed_to_answer(proposal),
```

Comment:
```suggestion
            answer_time: time_elapsed_to_answer(proposal),
```
I think this is more concise